catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/impl/ProcessingDetailsImpl.java

```diff
@@ -82,6 +93,21 @@ public ProcessingDetailsImpl(SourceProcessingDetails details, String sourceId) {
     return warnings;
   }
 
+  @Override
+  public boolean equals(Object o) {
+    if (!super.equals(o)) {
+      return false;
+    }
```

Comment:
✏️ Same here. It's fine for now. But it adds an unnecessary level of indirection.

Reply:
These lines call `SourceProcessingDetailsImpl`'s `equals()`. Do you think that adds an unnecessary level of indirection?

Reply:
No, you're right. I missed that. 👍